docs/user-guide/cli-install-configure-zosmf-security.md

```diff
@@ -0,0 +1,59 @@
+# Configuring z/OSMF Security
+
+Review the tasks that security administrators must complete to configure z/OSMF security for your Zowe CLI implementation.
+
+:::info Required role: security administrator
+:::
+
+## Configuring z/OS REST services SAF security
+
+A security administrator must configure security to allow z/OSMF System Authorization Facility (SAF) access to the resources that Zowe CLI requires. Zowe CLI uses REST endpoints that are associated with each z/OSMF REST API. After you complete all z/OSMF and z/OSMF cloud provisioning configurations, you can test your connection to z/OSMF to verify that Zowe CLI can communicate with z/OS systems.
+
+:::caution
+
+Before you allow users to issue z/OS console commands with Zowe CLI, security administrators should ensure that they provide access to commands that are appropriate for their organization.
+
+:::
+
+The following table details the required z/OSMF REST services and examples of the Zowe CLI features they enable. If the profile validation command returns any errors, use this table to find IBM documentation for the z/OSMF REST APIs.
```

Comment:
@zFernand0 - Is there a z/OSMF profile validation command? I don't know that this command exists, unless we are talking about `zowe zosmf check status`.